platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -1172,6 +1177,12 @@ fn main() {
                 .default_value("v0")
                 .help("Build for the given target architecture"),
         )
+        .arg(
+            Arg::new("optimize-size")
```

Comment:
nit: typically we use snake-case for the arg name
```suggestion
            Arg::new("optimize_size")
```